Ports/gmp/package.sh

```diff
@@ -2,6 +2,13 @@
 port='gmp'
 version='6.3.0'
 useconfigure='true'
+configopts=(
+   # We do not support 32-bit ABIs on 64-bit systems,
+   # and only support 64-bit systems
+   "ABI=64"
```

Comment:
This breaks on riscv64:
`configure: error: ABI=64 is not among the following valid choices: standard`